cmd/kubelet/app/server.go

```diff
@@ -166,7 +167,7 @@ func NewKubeletServer() *KubeletServer {
 		HealthzPort:                 10248,
 		HealthzBindAddress:          util.IP(net.ParseIP("127.0.0.1")),
 		RegisterNode:                true, // will be ignored if no apiserver is configured
-		OOMScoreAdj:                 -900,
+		OOMScoreAdj:                 -1000,
```

Comment:
I wouldn't change this, we do want the Kubelet to be killable if it is being a huge hog. -999?